int8      deviceNumber
bool      assistantRobot
bool      goHome
string    helpPose
int8      nOperations
string[]  pickPoses
string[]  placePoses